### DISCUSSION TITLE
特定のキーを押すと縦一列が反応する。

### DISCUSSION THREAD

**Discussion by poporirinu** [Yowkees/keyball#550] [Q&A] [answered]
お世話になっております
ダイオードの向きは確認済みです。
ProMicro左右交換も症状変わらず。

Jキーを押すとujmキー3つが反応し
OLEDにもujmと表記されます
対応策のご教授願います。![image](https://github.com/Yowkees/keyball/assets/160295919/ced1d2df-a1c3-4d4a-b986-54b26b209d4c)
![image](https://github.com/Yowkees/keyball/assets/160295919/12119bef-d965-4700-8857-2efc2c89c9ca)![image](https://github.com/Yowkees/keyball/assets/160295919/ff483071-8780-41c4-aa67-695e7e17d4a7)

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

添付しました写真の赤い丸で示した部分のはんだ付けが端子にうまく馴染んでいないように見えます。
はんだがしっかり馴染むようにはんだ付けをやり直してみてください。

![321270056-ced1d2df-a1c3-4d4a-b986-54b26b209d4c](https://github.com/Yowkees/keyball/assets/143698349/147c0ae1-35f3-459c-842c-ab9555ce3401)

**Reply by poporirinu** [reply]
ご回答ありがとうございます。
試しましたが改善されませんでした。
他に対応策はございますか？

**Reply by Ryo624** [reply]
ご対応ありがとうございます。

青い丸で示したキーソケットから少しはみ出しているはんだの除去してみるとどうなりますでしょうか。
![321513140-d1a2e565-df06-4964-9c01-8eaad16beced (2)](https://github.com/Yowkees/keyball/assets/143698349/04eec3e1-ded5-4472-b14f-fab6b2ad31af)

**Reply by poporirinu** [reply]
ありがとうございます、無事解決致しました。

別件ですが同キーボードにて2つトラブルが発生してしまいました。

1.左キーボードにUSB-cケーブルを刺すと
右キーボードとして認識される
![image](https://github.com/Yowkees/keyball/assets/160295919/14309db7-eb24-4966-9980-a6b5b1062d2f)![image](https://github.com/Yowkees/keyball/assets/160295919/a5db19a7-3d5d-45d6-a372-80432357846f)![image](https://github.com/Yowkees/keyball/assets/160295919/1957a369-ef71-4fe8-8583-649792cc465c)

2.右キーボードの黄色丸で囲んだキーが反応しない
![image](https://github.com/Yowkees/keyball/assets/160295919/38fc5827-9198-47a3-a18a-d1eb8cea16c6)
![image](https://github.com/Yowkees/keyball/assets/160295919/52d4c2d5-4fc6-454b-bc8e-e21db6da8105)
![image](https://github.com/Yowkees/keyball/assets/160295919/4ba5c05b-ce02-4e5e-8f2d-399c397ab96e)

**Reply by Ryo624** [reply, marked as answer]
ご対応ありがとうございます。
縦一列のキーは解決したようで良かったです。

1.左キーボードにUSB-cケーブルを刺すと右キーボードとして認識される
写真の青丸で示したProMicroのはんだ付けとLEFT判定のジャンパのはんだ付けをもう一度やり直していただいてもよろしいでしょう　　　か。
![322371145-14309db7-eb24-4966-9980-a6b5b1062d2f](https://github.com/Yowkees/keyball/assets/143698349/718c712b-6c43-4e20-984d-73ada38b3133)

2.右キーボードの黄色丸で囲んだキーが反応しない
ひとつのキーの不具合ですので、キーソケットもしくはダイオードが原因となっている可能性が高いので、はんだ付けをやり直してください。また、キーソケットの青い丸で示した部分が少しはんだがはみ出しているように見えるので除去できますでしょうか。
ダイオードはキーソケットのすぐ上のものが対応しているダイオードです。
![322371244-52d4c2d5-4fc6-454b-bc8e-e21db6da8105](https://github.com/Yowkees/keyball/assets/143698349/f59b8ea9-c131-4ed6-b6a5-15aa074dd493)

**Reply by poporirinu** [reply]
ご回答ありがとうございました。
全て解決致しました
今後ともよろしくお願い致します。

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます。
また何かありましたらご連絡ください。
